v4/core/cp4d_authenticator_test.go

```diff
@@ -372,6 +372,108 @@ func TestCp4dBackgroundTokenRefreshFailure(t *testing.T) {
 	assert.False(t, ok)
 }
 
+func TestCp4dBackgroundTokenRefreshIdle(t *testing.T) {
+	firstCall := true
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		if firstCall {
+			fmt.Fprintf(w, `{
+				"username":"hello",
+				"role":"user",
+				"permissions":[  
+					"administrator",
+					"deployment_admin"
+				],
+				"sub":"hello",
+				"iss":"John",
+				"aud":"DSX",
+				"uid":"999",
+				"accessToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImhlbGxvIiwicm9sZSI6InVzZXIiLCJwZXJtaXNzaW9ucyI6WyJhZG1pbmlzdHJhdG9yIiwiZGVwbG95bWVudF9hZG1pbiJdLCJzdWIiOiJoZWxsbyIsImlzcyI6IkpvaG4iLCJhdWQiOiJEU1giLCJ1aWQiOiI5OTkiLCJpYXQiOjE1NjAyNzcwNTEsImV4cCI6MTU2MDI4MTgxOSwianRpIjoiMDRkMjBiMjUtZWUyZC00MDBmLTg2MjMtOGNkODA3MGI1NDY4In0.cIodB4I6CCcX8vfIImz7Cytux3GpWyObt9Gkur5g1QI",
```

Comment:
It might be a little more readable to create a couple of local variables that hold the two different access tokens: `accessToken1` and `accessToken2` and use those wherever you're currently using string literals.